pkg/pillar/types/assignableadapters_test.go

```diff
@@ -462,9 +463,13 @@ func TestExpandControllers(t *testing.T) {
 	}
 }
 
-func TestΝvmeDeviceIsMountedByEVE(t *testing.T) {
+func TestNVMEIsUsed(t *testing.T) {
 	log := base.NewSourceLogObject(logrus.StandardLogger(), "test", 1234)
+	zfsManagerDir = "./zfsmanager"
 
+	if err := os.MkdirAll(zfsManagerDir, 0755); err != nil {
```

Comment:
I would recommend using [os.MkdirTemp](https://pkg.go.dev/os#MkdirTemp) instead (with "" as the first arg).